.bazelrc

```diff
@@ -381,10 +370,9 @@ build:release_windows --per_file_copt=.*capnp/rpc\.c++@/clang:-O1
 build:windows --cxxopt='/std:c++23preview' --host_cxxopt='/std:c++23preview'
 build:windows --copt='/D_CRT_USE_BUILTIN_OFFSETOF' --host_copt='/D_CRT_USE_BUILTIN_OFFSETOF'
 build:windows --copt='/DWIN32_LEAN_AND_MEAN' --host_copt='/DWIN32_LEAN_AND_MEAN'
-build:windows --copt='-Wno-microsoft-unqualified-friend' --host_copt='-Wno-microsoft-unqualified-friend'
 # The `/std:c++23preview` argument is unused during BoringSSL compilation and we don't
 # want a warning when compiling each file.
-build:windows --copt='-Wno-unused-command-line-argument' --host_copt='-Wno-unused-command-line-argument'
```

Comment:
Blanket disabling warnings about unused arguments is prone to covering up actual issues, in this case we weren't actually disabling exceptions on Windows when requested, missing out on some code size optimization potential (MSVC supports /GX- and clang-cl should support it based on the documentation, but the flag somehow still goes unused).